ajax/libs/datatables-fixedheader/package.json

```diff
@@ -19,5 +19,17 @@
   "author": {
     "name": "SpryMedia Ltd",
     "url": "https://datatables.net"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/DataTables/Dist-DataTables-FixedHeader.git",
+    "fileMap": [
+      {
+        "basePath": "js",
+        "files": [
+          "**/*"
```

Comment:
It's a little bit not safe or no sense to use **/* under a non-dist folder. We can use `dataTables.fixedHeader*` here.